hadoop-hdfs-project/hadoop-hdfs/src/site/markdown/WebHDFS.md

```diff
@@ -1604,6 +1605,26 @@ See also: [FileSystem](../../api/org/apache/hadoop/fs/FileSystem.html).renameSna
 
         {"SnapshotDiffReport":{"diffList":[],"fromSnapshot":"s3","snapshotRoot":"/foo","toSnapshot":"s4"}}
 
+### Get Snapshot Diff Iteratively
+
+* Submit a HTTP GET request.
+
+        curl -X GET curl -i -X GET "http://localhost:9870/webhdfs/v1/foo?op=GETSNAPSHOTDIFFLISTING&oldsnapshotname=s4&snapshotname=s5"
```

Comment:
good to mentioned that in the absence of `snapshotdiffstartpath` and `snapshotdiffindex`, their default values used will be empty string and -1 respectively.

Reply:
I added description about default values of the parameters.